redis-om-spring/src/main/java/com/redis/om/spring/repository/query/QueryUtils.java

```diff
@@ -3,19 +3,29 @@
 import java.util.Set;
 
 public class QueryUtils {
-  public static Set<Character> tagEscapeChars = Set.of( //
+  public static final Set<Character> TAG_ESCAPE_CHARS = Set.of( //
```

Comment:
IMHO these might generally be useful for JSON as well. Perhaps they belong elsewhere?

Reply:
I am using these QueryUtils across both JSON and Hash queries.